docs/wip_glassbox/wip-glassbox-customization.md

```diff
@@ -1,22 +1,5 @@
 # WIP Glassbox Customization
 
-## Forms
```

Comment:
a short description with a link to the full article should be left here (see Entity Extension)

Reply:
Okay added.